### PR TITLE
remove deprecated net.ipv4.ip_forward, use net.ipv4.conf.all.forwardi…

### LINKED ISSUE #784: CIS Feature: Deactivate Sysctl ICMP Redirect
**Background**:
PR #771 added the first iteration of the new `CIS` feature to Garden Linux. However, this feature is still declared as `WIP` and needs to be adjusted to fulfill all requirements of #647.

**Task**:
We need to ensure that ICMP redirects are disabled. While this is already configured and applied by `cis_os` the related test script `3.3.2_disable_icmp_redirerct` fails at least for `kvm-dev` images. This may be related to the QEMU guest agents, however we need to analyze this.

### PR DESCRIPTION
**Pull request by chrinorse**
…ng instead

**How to categorize this PR?**

/kind bug
/area os
/os garden-linux

**What this PR does / why we need it**:
remove deprecated net.ipv4.ip_forward, use net.ipv4.conf.all.forwarding instead

**Which issue(s) this PR fixes**:
Fixes #784 

**Special notes for your reviewer**:

**Release note**:

```feature user

```

### CODE REVIEW

**Review by gyptazy — APPROVED** [review]
lgtm

File: features/cloud/file.include/etc/sysctl.d/20-cloud.conf [hunk 1]
```diff
@@ -55,6 +55,3 @@ net.ipv4.conf.all.send_redirects = 1
 
 # Promote a secondary IP address in case a primary IP address is removed
 net.ipv4.conf.all.promote_secondaries = 1
-
-# Forward Packets between interfaces.
-net.ipv4.ip_forward = 1
```
Comment (MrBatschner) [inline features/cloud/file.include/etc/sysctl.d/20-cloud.conf:60]: No replacement for this directive?

**Review by MrBatschner — COMMENTED** [review]
see above

File: features/cloud/file.include/etc/sysctl.d/20-cloud.conf (hunk 1 above)
Comment (chrinorse) [reply features/cloud/file.include/etc/sysctl.d/20-cloud.conf:60]: No, in line 47 of the same file
```
net.ipv4.conf.all.forwarding = 1
```
is already set.

Comment (MrBatschner) [reply features/cloud/file.include/etc/sysctl.d/20-cloud.conf:60]: Next time, I will click on "Show all lines" before commenting. ;-)

**Pull request merged**